modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -274,7 +280,7 @@
 			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
 			return
```

Comment:
Also the whole `if(in_use)` thing should be at the beginning of the proc, after everything else that doesn't use it.

Reply:
this would need to be in another PR where I just clean up forge code.